testpilot/frontend/static-src/app/main.js

```diff
@@ -79,6 +79,15 @@ app.extend({
     } else {
       hitCallback();
     }
+  },
+
+  // Subscribe to basket
+  subscribeToBasket(email) {
+    const url = 'https://basket.mozilla.org/news/subscribe/';
+    const params = 'newsletters=test-pilot&email=' + email;
+    const request = new XMLHttpRequest();
+    request.open('POST', url);
+    request.send(params);
```

Comment:
We're using the `fetch()` api elsewhere, not sure if it makes sense to use here too.
https://github.com/mozilla/testpilot/blob/c4c41e893120132639347e41620fc84d054670f4/testpilot/frontend/static-src/app/views/settings-menu-view.js#L64